src/util/options.rs

```diff
@@ -259,11 +335,11 @@ options! {
     // Should we eagerly finish sweeping at the start of a collection? (not supported)
     eager_complete_sweep:  bool                 [env_var: true, command_line: true]  [always_valid] = false,
     // Should we ignore GCs requested by the user (e.g. java.lang.System.gc)?
-    ignore_system_g_c:     bool                 [env_var: true, command_line: true]  [always_valid] = false,
-    // The upper bound of nursery size.
-    max_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MAX_NURSERY,
-    // The lower bound of nusery size.
-    min_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MIN_NURSERY,
+    ignore_system_gc:      bool                 [env_var: true, command_line: true]  [always_valid] = false,
+    // FIXME: This is not a good way to have conflicting options -- we should refactor this
+    // The nursery size for generational plans. It can be one of Bounded or Fixed. The nursery size
+    // can be set like "Fixed:8192", for example, to have a Fixed nursery size of 8192 bytes
```

Comment:
Probably make it clear that for `Bounded`, the specified size is the upper bound.